### PR TITLE
libfabric 1.11.0

### PR DESCRIPTION
**Pull request by dtrodrigues**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/libfabric.rb
Comment (Rylan12) [inline Formula/libfabric.rb:6]:
```suggestion
  license ["BSD-2-Clause", "GPL-2.0-only"]
```

I don't see any indicator of `-or-later` so I think we need to go with `-only` (unless I missed something)

Comment (dtrodrigues) [reply Formula/libfabric.rb:6]: This is where I got th "or later" from: https://github.com/ofiwg/libfabric/blob/master/COPYING#L335-L338

Comment (Rylan12) [reply Formula/libfabric.rb:6]: I think that's just the part of the license text that explains that you are _allowed_ to use `-or-later`. Note that the lines you linked also appear in the [`GPL-2.0-only` license text](https://spdx.org/licenses/GPL-2.0-only.html#licenseText).

I think it's a similar situation to #59376

Comment (chenrui333) [reply Formula/libfabric.rb:6]: so when we cannot decide which `GPL-2.0-only` vs `GPL-2.0-or-later` can we simply go with `GPL-2.0` (even though that is deprecated license).

Comment (chenrui333) [reply Formula/libfabric.rb:6]: actually we can ask the maintainers about the license application in here, also.

Comment (chenrui333) [reply Formula/libfabric.rb:6]: @shefty, are you in the position who can clarify the license usage for the project? Thanks!

**Comment by dawidd6** [comment]
Remove the colon from commit message.

File: Formula/libfabric.rb
Comment (dtrodrigues) [reply Formula/libfabric.rb:6]: I think @Rylan12 is right, but should probably wait on an upstream maintainer to approve it. I'll push a change to be `GPL-2.0-only` and fix the commit message formatting.

Comment (Rylan12) [reply Formula/libfabric.rb]: > so when we cannot decide which `GPL-2.0-only` vs `GPL-2.0-or-later` can we simply go with `GPL-2.0` (even though that is deprecated license).

@chenrui333, in these cases, we should go with `GPL-2.0-only` as there is nothing that indicates that a later version can be used.

Comment (chenrui333) [reply Formula/libfabric.rb]: > > so when we cannot decide which `GPL-2.0-only` vs `GPL-2.0-or-later` can we simply go with `GPL-2.0` (even though that is deprecated license).
> 
> @chenrui333, in these cases, we should go with `GPL-2.0-only` as there is nothing that indicates that a later version can be used but we do know that `GPL-2.0` applies.

oh ok, good to know.
hmm, how can we possibly change the license in the future if the real intention is to have a `or-later` compatibility 🤔

Comment (Rylan12) [reply Formula/libfabric.rb]: Let's move forward with this. If a maintainer replies and indicates that we should use `GPL-2.0-or-later` we can open a PR to fix it.

**Review by Rylan12 — APPROVED** [review]

File: Formula/libfabric.rb
Comment (shefty) [reply Formula/libfabric.rb]: It should be GPU 2.0 only.  I will need to discuss the license with the organization to see if a later version is acceptable or not.

Comment (Rylan12) [reply Formula/libfabric.rb]: Thanks for the clarification, @shefty! Just to clarify: we are not asking for the license to be changed but just for clarification over which one is most correct. We will stick with `BSD-2-Clause` or `GPL-2.0-only`.

Comment (chenrui333) [reply Formula/libfabric.rb]: Thanks @shefty, that is helpful!

**Pull request closed without merging**
